standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1101,6 +1118,7 @@ struct CommitTxnRequest {
     5: optional CommitTxnKeyValue keyValue,
     6: optional bool exclWriteEnabled = true,
     7: optional TxnType txn_type,
+    8: optional set<AffectedRowCount> rowsAffected,
```

Comment:
if an older client connect - may it really leave this info out; and everything will work correctly?

Reply:
reverted this